testsuite/MDAnalysisTests/lib/test_util.py

```diff
@@ -545,18 +566,40 @@ def test_double_precision_box(self):
             residue_segindex=[0],
             trajectory=True,
             velocities=False,
-            forces=False)
+            forces=False,
+        )
         ts = u.trajectory.ts
         ts.frame = 0
         ts.dimensions = [10, 10, 10, 90, 90, 90]
         # assert ts.dimensions.dtype == np.float64
         # not applicable since #2213
-        ts.positions = np.array([[1, 1, 1, ], [9, 9, 9]], dtype=np.float32)
+        ts.positions = np.array(
+            [
```

Comment:
O no these are much worse

Reply:
Yep. See my comments above. I think it's the inconsistent trailing comma causing this.